From bc77c58977d66ce7e8d4358521b0ac6098b1a833 Mon Sep 17 00:00:00 2001
From: Xufeng Zhang <xufeng.zhang@windriver.com>
Date: Tue, 6 Aug 2013 15:13:57 +0800
Subject: [PATCH] arm/oprofile: Fix one compiling error

Since commit 7a47bae15(perf: Add context field to perf_event),
we add a context field to perf_event_create_kernel_counter(),
clearly this call in arm/oprofile has missed to be adjusted.

Signed-off-by: Xufeng Zhang <xufeng.zhang@windriver.com>
---
 arch/arm/oprofile/common.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/oprofile/common.c b/arch/arm/oprofile/common.c
index 11c6294..dab5417 100644
--- a/arch/arm/oprofile/common.c
+++ b/arch/arm/oprofile/common.c
@@ -159,7 +159,7 @@ static int op_create_counter(int cpu, int event)
 
 	pevent = perf_event_create_kernel_counter(&counter_config[event].attr,
 						  cpu, -1,
-						  op_overflow_handler);
+						  op_overflow_handler, NULL);
 
 	if (IS_ERR(pevent)) {
 		ret = PTR_ERR(pevent);
-- 
1.7.0

